Jenkinsfile

```diff
@@ -27,9 +27,27 @@ def deleteDirWin() {
     deleteDir()
 }
 
+def sourceCodePaths(){
+    // These paths will be passed to git diff
+    // If there are changes to them, CI/CD will continue else skip
+    def paths = ['src', 'make', 'stan', 'install-tbb.bat', 'makefile', 'runCmdStanTests.py', 'test-all.sh', 'Jenkinsfile']
```

Comment:
This is missing the lib folder and the examples folder. 

Instead of `src` this should be `src/test` and `src/cmdstan`. This would allow changes to `src/docs` to not trigger tests.